docs/projects/P3/index.md

```diff
@@ -0,0 +1,192 @@
+
+!!! note "Time Management"
+    Don’t spend too long during this exploration phase! Set deadlines within your team to ensure that you have enough time for both the design document and final integration deliverables described below.
+
+
+### Tools Checkpoint (10 pts)
+
+By the checkpoint deadline, your team should have decided on your **initial list of the N-1 tools** that you plan on exploring, and you must have **successfully installed and run at least one tool on your repository.**
+
+On Gradescope, submit your list of tools and a link to the PR that provides evidence to the tool that was successfully installed.
+
+### Tool Analysis Design Doc (60 pts)
+
+Create a Design Document/RFC that includes:
+
+- A **tools evaluations section** detailing your team’s analysis on each of the tools you experimented with
+- A **justification section** explaining which tool(s) you think the project should use moving forward
+- An **integration section** describing how the selected tool(s) shall be integrated into your process
+- A **conclusion section** summarizing your work
+
+Below, we provide more detailed instructions and page limit recommendations for each of the sections.
+
+#### Tool Evaluations (~N pages)
+
+For each of the N-1 tools explored by your team, you must provide:
+
+- Name and high-level description of what the tool does and a link to its documentation/source
+- Whether the tool is used for static or dynamic analysis
+- A link to the pull request made from the testing branch for this tool
+- A pro/con analysis of the tool and appropriate evidence to support your claims (i.e. screenshots). You can use the questions provided in the exploration phase above to shape your analysis.
+
+Each of these sections should take up approximately 1 page (including screenshots) and no more than 2 pages.
+
+#### Justification (~Half a page)
+
+After going through each of the tools, you should explicitly state the tool you are choosing to integrate for this project and provide a justification for why you are selecting this tool. You should refer to the pro/con analysis done in the prior sections and how they align with the goals of your team and the project overall.
+
+You must recommend at least one tool, even if it’s with reservations.
+
+#### Integration (<1 page)
+
+This section should address the different factors to take into consideration when integrating a new tool. At minimum, you should address the following:
+
+- **Technical Questions:** how are you integrating the tool (high-level)? At what point in the development/deployment process shall it be integrated? What sorts of customization or configuration will you be using?
+- **Developer Questions:** how should the team use this new tool within their development process?
+
+Your answers should be based on your experiences running the tools and, as much as possible, be grounded in data (such as tool usability, output, and customizability).
```

Comment:
I'm a little confused by this statement here - what are we trying to convey to the students here?

Reply:
I think this was just copied from the previous assignment writeup, basically making sure that they use evidence/supports their decisions with what they found in the research phase (i.e. "we saw that the linter flagged basically 80% of our files, most of which were false positives") rather than just saying some generic statements (i.e. "we felt like we should disable this feature")